### ISSUE TITLE
Mention all keybindings in help

### ISSUE THREAD

**Issue by jtdaugherty** [matterhorn-chat/matterhorn#139]
Right now, lots of catch-all keybindings don't get shown in the help output. Only the bindings that have explicit entries in our keybinding tables get listed. This makes things awkward because then the help output suggests there are fewer bindings than there actually are. So the task here is to augment our keybinding abstraction to allow for capturing those other keybindings.

This is also important because at some point we'll need a mechanism to allow keybindings to be changed, and that will also require a mechanism for calling out all known keybindings.

**Comment by jtdaugherty** [comment]
This includes new keybindings @aisamanra added for killing/navigating by words.

**Comment by aisamanra** [comment]
Those new keybindings are included as of 948bf52, because there's a separate `editingKeybindings` list that includes non-smart-quote-related keybindings.

**Comment by jtdaugherty** [comment]
I'm moving this out of the Public Release milestone since it's now blocked on the new customizable keybinding system task (which is not part of this milestone).
